pkg/subscriptions/subscriptions.go

```diff
@@ -212,7 +212,7 @@ func MountsWithUIDGID(mountLabel, containerRunDir, mountFile, mountPoint string,
 }
 
 func rchown(chowndir string, uid, gid int) error {
-	return filepath.Walk(chowndir, func(filePath string, f os.FileInfo, err error) error {
+	return filepath.Walk(chowndir, func(filePath string, _ os.FileInfo, err error) error {
```

Comment:
(I think this could use `filepath.WalkDir` for a tiny bit of performance optimization, OTOH it would still have a `_` unused parameter.)

Reply:
I rather not change the implementation here as it is unrelated to the linter.

Reply:
Sure.